specs/global/interfaces.json

```diff
@@ -0,0 +1,129 @@
+{
+    "Fingerprintable": {
+        "type": "object",
+        "properties": {
+            "labels": {
+                "$ref": "./schemas.json#/Labels"
+            },
+            "descriptions": {
+                "$ref": "./schemas.json#/Descriptions"
+            },
+            "aliases": {
+                "$ref": "./schemas.json#/Aliases"
+            }
+        },
+        "required": [
+            "labels",
+            "descriptions",
+            "aliases"
+        ]
+    },
+    "HashBearing": {
+        "type": "object",
+        "properties": { "hash": { "$ref": "#/SnakHash"} }
+    },
+    "HashedSnak": {
+        "allOf": [
+            {
+                "$ref": "#/Snak"
+            },
+            {
+                "type": "object",
+                "properties": { "hash": { "$ref": "#/SnakHash" } }
+            }
+        ]
+    },
+    "HashedSnakList": {
+        "type": "array",
+        "items": { "$ref": "#/HashedSnak" }
+    },
+    "HashedSnakMap": {
+        "type": "object",
+        "additionalProperties": {
+            "$ref": "#/HashedSnakList"
+        },
+        "example": {
+            "P31": [],
+            "P100": []
+        }
+    },
+    "MediawikiEditComment": { "type": "string" },
+    "MediawikiDeletionReason": { "type": "string" },
+    "MediawikiEditTags": {
+        "type": "array",
+        "items": { "$ref": "#/MediawikiEditTag" }
+    },
+    "MediawikiEditTag": { "type": "string" },
+    "SitelinkMap": {
+        "type": "object",
+        "additionalProperties": {
+            "$ref": "./schemas.json#/Sitelink"
+        },
+        "example": {
+            "afwiki": {
+                "site": "afwiki",
+                "title": "Douglas Adams",
+                "badges": [],
+                "url": "https://af.wikipedia.org/wiki/Douglas_Adams"
+            },
+            "arwiki": {
+                "site": "arwiki",
+                "title": "دوغلاس آدمز",
+                "badges": [],
+                "url": "https://ar.wikipedia.org/wiki/%D8%AF%D9%88%D8%BA%D9%84%D8%A7%D8%B3_%D8%A2%D8%AF%D9%85%D8%B2"
+            }
+        }
+    },
+    "Snak": {
```

Comment:
Should we maybe still Include Snak as part of the shown Schmas on the specs? (It is used quite frequently). What do you think @addshore, @wiese

Reply:
tl;dr I'd say yes.

I'm torn about this. Snaks are a core concept and can do with some space where it is acknowledged that they exist and how they work (c.f. https://www.mediawiki.org/wiki/Wikibase/DataModel#Snaks) yet they don't really have a life of their own. But we could say the same thing about `Term` - or this one needs to move out as well.

Reply:
That was my instinct as well, and as adam already answered, term is uses, so why not snak. In this case I will move it. thank you for the clear response!